test/lib/linter/rules/snapshots/XmlInJs.ts.md

```diff
@@ -231,6 +255,30 @@ Generated by [AVA](https://avajs.dev).
             ruleId: 'no-deprecated-api',
             severity: 2,
           },
+          {
+            column: 23,
+            line: 32,
+            message: 'Use of deprecated property \'xmlview\' (sap.ui.xmlview)',
+            messageDetails: 'Deprecated test message',
+            ruleId: 'no-deprecated-api',
+            severity: 2,
+          },
+          {
+            column: 17,
+            line: 33,
+            message: 'Call to deprecated function \'sapUiXmlView\'',
```

Comment:
The deprecation is correct, but to me this seems a bit missleading. There's no such thing as `sapUiXmlView`, but its assignment. While in the example, the solution is obvious, in real code this might not be the case and might lead to ambiguous situations

Reply:
We might try to enhance this in general by adding the original API name to the message. This is sometimes working, but I think in this case it does not, as the `additionalMessage` is empty.